scenarios/command/c/command_client/main.c

```diff
@@ -0,0 +1,205 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <uuid/uuid.h>
+
+#include "mosquitto.h"
+#include "mqtt_callbacks.h"
+#include "mqtt_protocol.h"
+#include "mqtt_setup.h"
+
+#define PAYLOAD "\n\v\b\261Í\244\006\020\364\254\265d\022\nmobile-app"
+#define RESPONSE_TOPIC "vehicles/vehicle03/command/unlock/response"
+#define PUB_TOPIC "vehicles/vehicle03/command/unlock/request"
+#define COMMAND_CONTENT_TYPE "application/protobuf"
+#define COMMAND_TIMEOUT_SEC 10
+
+#define QOS 1
```

Comment:
```suggestion
#define QOS_AT_LEAST_ONCE 1
```

Reply:
This is defining QOS for this sample - while we're only really supporting 1 now, in the future the expectation is that the customer could potentially set this to 0, 1, or 2 depending on the sample

Reply:
QOS_LEVEL ?

Reply:
Resolved in all samples